sdk/js/src/utils/consts.ts

```diff
@@ -40,6 +40,9 @@ export const CHAINS = {
   neutron: 4003,
   celestia: 4004,
   sepolia: 10002,
+  arbitrum_sepolia: 10003,
+  base_sepolia: 10004,
+  optimism_sepolia: 10005,
```

Comment:
These are the only chains that have a separator. Hopefully nobody is doing some `string.split("_")` or regex that would start failing w this

Reply:
also, for numeric chain ids, I'm guessing sepolia is `10002` because eth is `2` 

If we want to carry that pattern forward (do we?) we should make these `10023`,`10030`,`10024`

Reply:
Actually, that's not how Sepolia got to be 10002. That was just the next number after Ropsten, which was 10001. These are just sequentially assigned numbers. I don't think a scheme like you suggest would work, because in a year or two, there will probably be a Sepolia replacement, so we'll have to do this again. I think the chain ID is just a number.

Reply:
Should have made this 2 separate comments, unresolved since the underscore in name is still in question

Reply:
We discussed underscore vs. no underscore offline with the team, and the consensus was that the underscore was okay.